judo-runtime-core-bootstrap/src/main/java/hu/blackbelt/judo/runtime/core/bootstrap/dispatcher/DispatcherFunctionProviderProvider.java

```diff
@@ -34,10 +34,12 @@ public class DispatcherFunctionProviderProvider implements Provider<DispatcherFu
     @Override
     public DispatcherFunctionProvider get() {
         final EMap<EOperation, Function<Payload, Payload>> scripts = new BasicEMap<>();
+        final EMap<EOperation, Function<Payload, Payload>> sdkFunctions = new BasicEMap<>();
+
         DispatcherFunctionProvider dispatcherFunctionProvider = new DispatcherFunctionProvider() {
             @Override
             public EMap<EOperation, Function<Payload, Payload>> getSdkFunctions() {
-                return ECollections.emptyEMap();
+                return sdkFunctions;
```

Comment:
Why do we need a dedicated variable if it's only used here and nowhere else?

Reply:
It is used to add new function. You get the list and add your function.